### DISCUSSION TITLE
How do I make Lane width bigger on mobile and can you add four separate tiles on screen for mobile?

### DISCUSSION THREAD

**Discussion by BladeNothing** [ppy/osu#25095] [Q&A] [answered]
I'd l like to say that I wouldn't call this a bug, I just didn't know what to categorize this issue (maybe) I've been having.

So for starters I'm a mobile Lazer player, I know I can just install Malody and get basically the same experience to Osu!mania or play Rosu!mania (ROBLOX version of Osu!mania) or Funky Friday (and on malody there is no storyboard and video, on the last 2 you can't install any beat maps of your choice and I just prefer Osu mania also because 2 of my favorite maps Happen to be storyboard). 

Anyways, the issues I've been having with trying to play on mobile are that the Lanes are too small even with the default skin, but that wouldn't even matter because I can't import files of any into Lazer (it just opens the file up in the section I've opened it on). I've tried solving this issue by changing UI Scale, turning off the shrink game, Changing my Smallest Screen width from 550 to 440  to 400 (a little under my default screen size). 

Also I realized that for mobile you need to tap the keys on the lane...and I just so happen to be an up scroll player not a downscroll, but the Lane is too small for my fingers to touch anyways. 

Could you do like what FF, Malody, and Rosu!Mania do? Like Four tiles on the screen that you need to click? Or could you add an extra setting for it? I put an image of me in Funky Friday to show what I mean by Four Tiles, A video to show that I just can't import Beatmaps even when I long press (since The browsing feature doesn't include as many maps as the website does) it just opens up the Beatmap on the section I was in. Another problem is that they also don't appear in the downloads section and I needed to go to Malody a game which I could just import Beatmaps just fine, and it still wouldn't let me import. And I also put 2 images of me playing a song in game with different Screen widths (should look the same as my point is that they remain the same, which is small).

This isn't what I'd call an issue, it's more like "I don't know how to adjust these things on mobile or if I even can". I would very much appreciate it if you could help me out with this, pretty ples with cherry cream on top 🥺

![Screenshot_20231011-201848](https://github.com/ppy/osu/assets/147674482/bdfc8599-4b34-4294-88df-ff3e6c5518c7)
![Screenshot_20231011-201508](https://github.com/ppy/osu/assets/147674482/5b15c7ca-adaa-4ae1-98bb-6c2f836cf476)
![Screenshot_20231011-201206](https://github.com/ppy/osu/assets/147674482/bb606d20-86e0-43ed-bb4e-9ccff04f9487)

(Video can't be added so just take my word for the Beatmap opening up instead of being selected)

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/23377

**Reply by BladeNothing** [reply]
Thanks Peppy.
